force-app/main/default/objects/Session_Registration__c/validationRules/Is_Overlaps_Check.validationRule-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Is_Overlaps_Check</fullName>
+    <active>true</active>
+    <errorConditionFormula>IsOverlaps__c  = TRUE</errorConditionFormula>
```

Comment:
For the condition, you don't need to check if IsOverlaps__c is equal to TRUE. "IsOverlaps__c" should be enough for the condition. @bmtemakina